airmozilla/main/templates/main/event.html

```diff
@@ -135,15 +135,29 @@ <h1 class="entry-title">
 
       {% if tags %}
         <div class="tab">
+         <input type="radio" id="tab-2" name="tab-group-1">
+         <label for="tab-2">Tags</label>
+         <div class="content">
+           <p>
+             {% for tag in tags %}
+               <a href="{{ url('main:home') }}?tag={{ tag | urlencode }}" class="tag">{{ tag }}</a>{% if not loop.last %}, {% endif %}
+             {% endfor %}
+             {% if request.user.is_active %}
+              -<a href="{{ url('manage:event_edit', event.pk) }}" class="tag"> You can edit this here</a>
+             {% endif %}
+           </p>
+         </div>
+       </div>
+       {% elif not tags and request.user.is_active %}
+         <div class="tab">
           <input type="radio" id="tab-2" name="tab-group-1">
           <label for="tab-2">Tags</label>
           <div class="content">
-            <p>
-              {% for tag in tags %}
-                <a href="{{ url('main:home') }}?tag={{ tag | urlencode }}" class="tag">{{ tag }}</a>{% if not loop.last %}, {% endif %}
-              {% endfor %}
-            </p>
-          </div>
+          <p>
+          {{ _('Tags') }}:
+            <a href="{{ url('manage:event_edit', event.pk) }}" class="tag">Hey help us assign some tags for this event</a>
```

Comment:
Nice text!
The URL is wrong. That takes you to the management version. I think the correct link is `main:event_edit`